wiki/superblock.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Superblocks
+layout: wiki
+description: Explains what a superblock is, how they get made, and what exactly is inside one
+---
+
+# Overview
+
+A superblock is a special block that stores information about how much work everyone 
+in the network has done in [BOINC](boinc "wikilink"). When a superblock happens, [magnitude](magnitude "wikilink")
+gets updated. Superblocks happen almost exactly every 24 hours.
+
+Anyone can [stake](staking "wikilink") a superblock (if they are staking). If you 
+end up doing this by chance, you will see this icon
+![superblock icon](/assets/img/wiki/superblock.png){:height="44px"}. {% comment %} display the icon at the size of about two lines  {% endcomment %}
+Note that it doesn't really change anything for you as a staker other than just seeing a different icon
+
+
+# How Superblocks Get Made 
+
+1. A previous superblock is made and a 24 hour clock starts for the next superblock to be due[^1]
+2. Stats are independently gathered by the [scrapers](scrapers "wikilink") from the BOINC projects
+3. Those stats are sent out to each wallet which compare them and check that they match. 
+If at least 60% (rounded up) of the scrapers exactly match[^2], a superblock can be made (when it's due)
+4. Someone stakes and they include the new data from the scrapers
+
+Note that this is just a brief overview and there are many security and other mechanisms left out in the description
```

Comment:
"there are many security and other mechanisms" sounds awkward

Reply:
Change it to "Note that this is just a brief overview and there are many security mechanisms and other parts left out in the description"